### ISSUE TITLE
static reports wrong mime if 304

### ISSUE THREAD

**Issue by dvv** [creationix/creationix#2]
Hi! After https://github.com/creationix/creationix/blob/master/static.js#L52 should go Content-Type:, or subj occurs.
Best regards,
--Vladimir

**Comment by creationix** [comment]
I don't understand.  You're saying that content-type is required for 304 responses?

**Comment by dvv** [comment]
Yes. At least in chrome I get "Resource interpreted as script but transferred with MIME type text/plain." warnings for JS files. I guess 304 just instructs to hit the cache, but mime of cached file is unknown.
